docs/topics_5/web-running-saved-analysis.adoc

```diff
@@ -0,0 +1,16 @@
+
+
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='running_saved_analysis_{context}']
+= Running a saved analysis
+
+You can run an analysis of the last saved version of a project from the *Analysis results* screen.
```

Comment:
I thought we agreed that the user is not running a project. The user is running an analysis.

Reply:
Also, the "Analysis results" screen clause does not add anything. I think you should take out "from..."

Reply:
I did not write "You can run a project..." I wrote "You can run an analysis...". But, I am changing this to "You can run a saved analysis." in line with your comments.